### ISSUE TITLE
Improving Release Notes

### ISSUE THREAD

**Issue by slackmoehrle** [cocos2d/cocos2d-x-docs#14]
I would like to improve our release notes that we send out to developers. UnReal does an amazing job, example: https://www.unrealengine.com/en-US/blog/unreal-engine-4-17-released

What do you think about improving the quality of the release notes with more graphics and videos? (if the features make sense for these extras).

**Comment by minggo** [comment]
It is better if we can provide more graphics and videos. Did you mean we should put release note here?

**Comment by slackmoehrle** [comment]
@minggo Yes, I think since they are static after releasing we should make a section and start adding v3.15, v3.16 +.

**Comment by minggo** [comment]
k, sounds reasonable
